app/views/merchants/items/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @merchant.name %></h2> 
+
+<h3>New Item</h3>
+
+<%= form_with model: [@merchant, @item], local: true do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name, required: true %>
+  <%= f.label :description, required: true  %>
+  <%= f.text_area :description, required: true  %>
+  <%= f.label :unit_price %>
+  <%= f.number_field :unit_price,  in: 0.0..99999999.0, step: 0.01, value: sprintf("%.2f", 0.0), required: true  %>
+  <%= f.label :enabled, 'Enabled?' %>
+  <%= f.check_box :enabled, {checked: false} %>
```

Comment:
Great work on the check box for the enabled/disabled & on the unit_price stuff!!!